app/proxy/service_test.go

```diff
@@ -116,7 +116,29 @@ func TestCallResolve(t *testing.T) {
 	rawCallReponse := c.Call(request)
 	parseRawResponse(t, rawCallReponse, &resolveResponse)
 	assert.Equal(t, resolvedClaimID, resolveResponse[resolvedURL].ClaimID)
-	assert.True(t, svc.GetExecTimeMetrics("resolve").ExecTime > 0)
+	assert.True(t, svc.GetMetricsValue("resolve").Value > 0)
+}
+
+func TestCallAccountBalance(t *testing.T) {
+	// TODO: Add actual account balance response check after 0.39 support is added to lbry.go
+	// var accountBalanceResponse ljsonrpc.AccountBalanceResponse
+
+	rand.Seed(time.Now().UnixNano())
+	dummyAccountID := rand.Int()
+
+	acc, _ := lbrynet.CreateAccount(dummyAccountID)
+	defer lbrynet.RemoveAccount(dummyAccountID)
+
+	svc := NewService(config.GetLbrynet())
+	c := svc.NewCaller()
+	c.SetAccountID(acc.ID)
+
+	request := newRawRequest(t, "account_balance", nil)
+	hook := logrus_test.NewLocal(svc.logger.Logger())
+	c.Call(request)
+
+	assert.Equal(t, map[string]interface{}{"account_id": "****"}, hook.LastEntry().Data["params"])
```

Comment:
what is the meaning of `***`? Is that a default account of sorts? I would probably put this into a constant so the code reads better.

Reply:
That's just a masked value so we see the field is there but the value is not revealed. It is defined as a constant in the code, just double-checking its value here in the test.